### ISSUE TITLE
Incorrect condition tree in i6_core/returnn/oggzip.py

### ISSUE THREAD

**Issue by Max-Ryujin** [rwth-i6/i6_core#485]
The if statements in the main `run `function do not allow argument combinations. 
It is unclear why this should be especially why they all should be mutually exclusive.

**Comment by vieting** [comment]
To add a bit of context: This is about the `BlissToOggZipJob`. When building the call to RETURNN, we have
```
        if self.no_audio:
            args.extend(["--no_ogg"])
        elif self.no_conversion:
            args.extend(["--no_conversion"])
        elif self.ffmpeg_acodec:
            args.extend(["--ffmpeg_acodec", self.ffmpeg_acodec])
        else:
            if self.rasr_cache is not None:
                args.extend(["--sprint_cache", tk.uncached_path(self.rasr_cache)])
            if self.raw_sample_rate is not None:
                args.extend(["--raw_sample_rate", str(self.raw_sample_rate)])
            if self.feat_sample_rate is not None:
                args.extend(["--feat_sample_rate", str(self.feat_sample_rate)])
```
I don't think the synchronization with a RASR cache belongs into the `else` branch, it should be independent of `no_audio`, `no_conversion` and `ffmpeg_acodec`, right?

Maybe @JackTemaki you have an opinion about this?

**Comment by michelwi** [comment]
> It is unclear why this should be especially why they all should be mutually exclusive.

I think they imply each other in the order of appearance: 
* If there is no audio, then there cannot be any conversion of audio files
* if we do not convert any audio file, then we do not need any ffmpeg codecs

> synchronization with a RASR cache 

I can see that if we do not have any audio, then synchronization with rasr is pointless. but otherwise I would think that synchronization should be an option independent of conversion and/or codec

**Comment by JackTemaki** [comment]
Synchronization with RASR means you can not do a direct file pass-through and you need audio, so except for the last, the ffmpeg_acodec, this is correct from my point of view.

**Comment by vieting** [comment]
As just discussed offline with @JackTemaki: I misinterpreted the option `no_conversion` as "no conversion of the audio format" while it actually means "no modification of audio at all, file pass-through". Therefore, it is valid that we have to set `no_conversion=False` when using a RASR cache.

**Comment by JackTemaki** [comment]
resolved by #486
